### PR TITLE
Add ability to develop Fiddle on macOS.

### PR DESCRIPTION
**Pull request by saeta**
Because SeqIO doesn't work on mac, this change tweaks `setup.py` to not install it, and to skip tests in `tf_test.py` and `seqio_test.py` if `tf`/`seqio` (respectively) are not installed.

### CODE REVIEW

**Review by dhr — CHANGES_REQUESTED** [review]
Thanks! Looks good, a few quick thoughts.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ https://github.com/$$YOUR_USERNAME$$/fiddle`.
 
 ### Python dependencies
 
+> Note: if you're developing on a mac, you'll need xcode installed. (Required for PyType.)
```
[the remark this answers is not in the document]
Comment (saeta) [reply CONTRIBUTING.md]: Great catch. Thanks! Fixed.

File: fiddle/extensions/tf_test.py [hunk 2]
```diff
@@ -26,9 +26,23 @@
...
-import libcst as cst
-import tensorflow as tf
+
+try:
+  import fiddle.extensions.tf
+  import libcst as cst
+  import tensorflow as tf
+  TF_AVAILABLE = True
+except ImportError:
+  TF_AVAILABLE = False
```
[the remark this answers is not in the document]
Comment (saeta) [reply fiddle/extensions/tf_test.py]: Good idea. I'm fine with either approach (either the command line flags or auto-ignoring). I guess disabling via the command line flags (while more annoying for mac-based developers) is safer to ensure that something doesn't accidentally break silently. Happy to roll forward to the approach you suggest in a future PR should we decide.

[the remark this answers is not in the document]
Comment (dhr) [reply fiddle/extensions/tf_test.py]: Sounds good, using flags seems simplest to me.

**Pull request merged**